org_onebusaway_iphone_Prefix.pch

```diff
@@ -13,11 +13,11 @@
     #import "OBACommonV1.h"
     #import "OBACommon.h"
 
-    #import "TestFlight.h"
     #import "GAI.h"
     #import "GAIFields.h"
     #import "GAITracker.h"
     #import "GAIDictionaryBuilder.h"
+    #define TFLog NSLog
```

Comment:
Here is what I was talking about... do we need `TFLog` anymore since TestFlight has been removed?